.env.qa

```diff
@@ -15,4 +15,5 @@ REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/billing-api
 REACT_APP_MANUAL_STATUS_ENABLED=true
 REACT_APP_MANUAL_STATUS_FILE_URL = https://cdn.fromdoppler.com/webapp-status.json
 REACT_APP_DOPPLER_CONTACT_POLICY_URL=https://apisqa.fromdoppler.net/contact-policies
-REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/doppler-users-api
\ No newline at end of file
+REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/doppler-users-api
+REACT_APP_STATIC_DATA_BASE_URL = https://cdn.fromdoppler.com/static-data
```

Comment:
Why would you add this url as an environment variable? This is not changing now.

Reply:
Do you say that is enough add this variable only in ```.env```?

Reply:
The  below review answered my question. I removed this url from all environment variables.